source/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -25,7 +25,7 @@ export default Component.extend({
   },
 
   titles: computed('todos.[]', function() {
-    let todos = this.get('todos');
+    let todos = this.todos;
     return todos.mapBy('title');
```

Comment:
like above, these can now be simplified to e.g. `return this.todos.mapBy('title')`